x-pack/libbeat/cmd/enroll.go

```diff
@@ -75,7 +79,7 @@ func genEnrollCmd(name, version string) *cobra.Command {
 				}
 				enrollmentToken, err = client.CreateEnrollmentToken()
 				if err != nil {
-					return errors.Wrap(err, "Creating a new enrollment token")
+					return errors.Wrap(err, "Error creating a new enrollment token")
```

Comment:
should be lower case?

Reply:
These are shown to the user (command line) so I think it's better to keep them upper case